plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/ThriftHiveMetastore.java

```diff
@@ -1062,10 +1062,11 @@ public void createTable(Table table)
     @Override
     public void dropTable(String databaseName, String tableName, boolean deleteData)
```

Comment:
`Ignore internal TableNotFoundException in DROP TABLE for ThriftHiveMetastore` -> `Ignore TableNotFoundException on retries when dropping the table`